addons/binding/org.openhab.binding.onebusaway/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onebusaway"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-group-type id="arrivalRange">
+        <label>Range</label>
+        <description>Range for a route event</description>
+        <channels>
+            <channel id="arrival" typeId="arrival" />
+            <channel id="departure" typeId="departure" />
+            <channel id="event" typeId="routeEvent" />
+            <channel id="update" typeId="update" />
+        </channels>
+    </channel-group-type>
+
+    <channel-type id="arrival">
+        <item-type>DateTime</item-type>
+        <label>Arrival Time</label>
+        <description>The arrival time of the event</description>
+        <state readOnly="true" pattern="%1$tF %1$tR" />
+        <config-description-ref uri="channel-type:onebusaway:config" />
+    </channel-type>
+
+    <channel-type id="departure">
+        <item-type>DateTime</item-type>
+        <label>Departure Time</label>
+        <description>The departure time of the event</description>
+        <state readOnly="true" pattern="%1$tF %1$tR" />
+        <config-description-ref uri="channel-type:onebusaway:config" />
+    </channel-type>
+
+    <channel-type id="routeEvent">
+        <kind>trigger</kind>
+        <label>Route event</label>
+        <description>Route event</description>
+        <event>
+            <options>
+                <option value="ARRIVAL">arrival</option>
+                <option value="DEPARTURE">departure</option>
+            </options>
+        </event>
+        <config-description-ref uri="channel-type:onebusaway:config" />
+    </channel-type>
+
+    <channel-type id="update">
+        <item-type>DateTime</item-type>
+        <label>Last Update Time</label>
+        <description>The last time information was updated</description>
+        <state readOnly="true" pattern="%1$tF %1$tR" />
+        <config-description-ref uri="channel-type:onebusaway:config" />
```

Comment:
>  to write a rule that runs if, say, the information was last updated

This is a feature that should be part of the rule engine (this is what https://github.com/eclipse/smarthome/pull/3241 tries to do), not of (every single) channel - so I would strongly suggest to remove it here.